mail-client/claws-mail/claws-mail-4.1.1-r1.ebuild

```diff
@@ -0,0 +1,217 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{8..11} )
+
+inherit autotools desktop python-any-r1 xdg
+
+DESCRIPTION="An email client (and news reader) based on GTK+"
+HOMEPAGE="https://www.claws-mail.org/"
+
+if [[ "${PV}" == *9999 ]] ; then
+	inherit git-r3
+	EGIT_REPO_URI="https://git.claws-mail.org/readonly/claws.git"
+else
+	SRC_URI="https://www.claws-mail.org/download.php?file=releases/${P}.tar.xz"
+	KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ppc ~ppc64 ~riscv ~sparc ~x86"
+fi
+
+SLOT="0"
+QA_PKGCONFIG_VERSION="${PV}.0"
+LICENSE="GPL-3"
+
+IUSE="archive bogofilter calendar clamav dbus debug dillo doc gdata +gnutls +oauth +imap ldap +libcanberra +libnotify litehtml networkmanager nls nntp +notification pdf perl +pgp rss session sieve smime spamassassin spam-report spell startup-notification svg valgrind webkit xface"
```

Comment:
Let's keep the USE flags sorted (especially here with all the possible ones) - "+oauth" should be added just before "pdf"